tomcat/build.go

```diff
@@ -123,8 +124,13 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 
 	var externalConfigurationDependency *libpak.BuildpackDependency
 	if uri, ok := cr.Resolve("BP_TOMCAT_EXT_CONF_URI"); ok {
-		v, _ := cr.Resolve("BP_TOMCAT_EXT_CONF_VERSION")
-		s, _ := cr.Resolve("BP_TOMCAT_EXT_CONF_SHA256")
+		v, versionExists := cr.Resolve("BP_TOMCAT_EXT_CONF_VERSION")
+		s, shaExists := cr.Resolve("BP_TOMCAT_EXT_CONF_SHA256")
+
+		if !versionExists && !shaExists {
+			v = time.Now().Format(time.RFC3339)
+			b.Logger.Infof("No BP_TOMCAT_EXT_CONF_VERSION or BP_TOMCAT_EXT_CONF_SHA256 provided, using version '%s'", v)
```

Comment:
```suggestion
			b.Logger.Infof(color.YellowString("WARNING: No BP_TOMCAT_EXT_CONF_VERSION or BP_TOMCAT_EXT_CONF_SHA256 provided, so no layer caching will occur."))
```

I don't mind putting a timestamp in the version to force it to update, but I think we should try to make that an implementation detail and not show it to the user.